sycl/doc/extensions/InvokeSIMD/InvokeSIMD.asciidoc

```diff
@@ -385,6 +384,24 @@ Since this execution model guarantees SIMD-like behavior, there is no need for
 the user to insert any form of explicit synchronization functions to ensure
 memory consistency across SIMD lanes.
 
+Due to potential differences in execution model, a SIMD function cannot call
```

Comment:
I think this is a bit unclear. Let's say we have a SIMD function `foo` in `foo.cpp` and a `SYCL_EXTERNAL` function `bar`. It is obvious that this sentence prevents calling `bar` if `foo.cpp` only contains a declaration but not the definition. I think it is unclear whether this also prevents calling `bar` if `foo.cpp` also contains the definition of `bar`. In the later case `SYCL_EXTERNAL` isn't needed (but it might still be on the function because the function is called from other TUs). And it is unclear whether the intention is that even in that case calling the function is not allowed.